apps/glusterfs/models.go

```diff
@@ -25,6 +25,25 @@ import (
 	"sort"
 )
 
+//Topology
+
+type Node struct {
+	NodeInfo NodeInfoResponse `json:"nodeinfo"`
+}
+
+type Nodes struct {
+	Nodes []Node
+}
+
+type Cluster struct {
+	Id       string `json:"id"`
+	NodeList Nodes  `json:"nodes"`
+}
+
+type Topo struct {
+	ClusterList []Cluster `json:"clusters"`
+}
+
```

Comment:
These should only be in the server models.go directory if they are used for REST commands.  Instead, create a models file for the heketi-cli